e2e-tests/support/tasks.js

```diff
@@ -93,7 +93,7 @@ module.exports = {
     /**
      * Inserts utilisation report details to the SQL database
      * @param {Partial<import('@ukef/dtfs2-common').UtilisationReportEntity[]>} utilisationReports
-     * @returns {import('@ukef/dtfs2-common').UtilisationReportEntity[]} The inserted reports
+     * @returns {Promise<import('@ukef/dtfs2-common').UtilisationReportEntity[]>} The inserted reports
```

Comment:
I _think_ that because we actually import `UtilisationReportEntity`, we should be able to remove the `import('@ukef/dtfs2-common')`